kairon/api/app/routers/history.py

```diff
@@ -21,212 +21,278 @@
 
 
 @router.get("/users", response_model=Response)
-async def chat_history_users(month: int = Query(default=1, ge=1, le=6), current_user: User = Security(Authentication.get_current_user_and_bot, scopes=TESTER_ACCESS)):
+async def chat_history_users(
+        from_date: datetime.date = datetime.date.today() - datetime.timedelta(30),
```

Comment:
will date.today() return value in utc?
if not, shouldnt we be using datetime.utcnow().date() instead?